cli/lib/kontena/client.rb

```diff
@@ -55,6 +55,7 @@ def initialize(api_url, token = nil, options = {})
         connect_timeout: ENV["EXCON_CONNECT_TIMEOUT"] ? ENV["EXCON_CONNECT_TIMEOUT"].to_i : 5,
         read_timeout:    ENV["EXCON_READ_TIMEOUT"]    ? ENV["EXCON_READ_TIMEOUT"].to_i    : 30,
         write_timeout:   ENV["EXCON_WRITE_TIMEOUT"]   ? ENV["EXCON_WRITE_TIMEOUT"].to_i   : 5,
+        ssl_ca_path:     ENV["EXCON_SSL_CA_PATH"]     ? ENV["EXCON_SSL_CA_PATH"] : "/etc/ssl/certs",
```

Comment:
Nice that worked :) Didnt find anything about this in the documentation or the slack channel... so you can drop the pull request. Maybe you can add this "as tip" to the protip when displaying the error message ;)